php_apcu/README.md

```diff
@@ -0,0 +1,80 @@
+# Agent Check: php_apcu
+
+## Overview
+
+This check monitors [php_apcu][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the php_apcu check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build php_apcu` to build the package.
+
+3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/php_apcu/dist/<ARTIFACT_NAME>.whl`.
+
+#### APCu Setup
+
+Provide a URL to the collection program.
+
+APCu does not have a metric output by itself.
+
+* /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+
+It will work if you refer to it as a php file, but you may want to configure it as follows
+
+```
+Alias /apcu-status /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+<Location /apcu-status>
+    Require all denied
+    Require local
+</Location>
+```
+
```

Comment:
and add configure it in the `conf.yaml` file:
```
instances
  - url: http://localhost/apcu-status
```

Reply:
already going to provide that URL.
https://github.com/withgod/integrations-extras/blob/feature/apcu/php_apcu/assets/configuration/spec.yaml#L11-L27
https://github.com/withgod/integrations-extras/blob/feature/apcu/php_apcu/datadog_checks/php_apcu/data/conf.yaml.example#L61